docs/reference/release-notes/highlights.asciidoc

```diff
@@ -37,4 +37,90 @@ Other versions:
 // [discrete]
 // === Heading
 //
-// Description.
\ No newline at end of file
+// Description.
+
+// tag::notable-highlights[]
+
+[discrete]
+[[frozen-tier-ga]]
+=== Frozen tier is now GA
+
+The frozen tier and shared-cache searchable snapshots are now generally
+available. These features were introduced as technical previews in 7.12. See
+the
+{ref-bare}/7.12/release-highlights.html#frozen-tier-shared-snapshot-cache[7.12
+announcement].
+
+[discrete]
+[[index-runtime-fields]]
+=== Index runtime fields
+
+Runtime fields let you dynamically create fields from other fields and document
+properties at query time. These query-time runtime fields optimize for
+flexibility over speed, letting you change them at any time.
+
+With {minor-version}, you can now also create index-time runtime fields, trading
+flexibility for better query performance. See {ref}/runtime-indexed.html[Index a
+runtime field].
+
+[discrete]
+[[match-ipv4-ipv6-addresses-against-cidr-ranges-painless]]
+=== Match IPv4 and IPv6 addresses against CIDR ranges in Painless
+
+Painless expressions can now match IPv4 and IPv6 addresses against CIDR ranges.
+This is useful for classifying and grouping IP addresses in security and
+monitoring use cases. This feature is available in all contexts, including
+runtime fields, ingest, search, and aggregations.
+
+For example:
+
+[source,painless]
+----
+CIDR myCidrRange = new CIDR('10.1.1.0/25');
+myCidrRange.contains('10.1.1.127'); // true
+myCidrRange.contains('10.1.1.129'); // false
+
+myCidrRange = new CIDR('2001:0db8:85a3::/64');
+myCidrRange.contains('2001:0db8:85a3:0000:0000:8a2e:0370:7334'); // true
+myCidrRange.contains('2001:0db8:85a3:0001:0000:8a2e:0370:7334'); // false
+----
+
+[discrete]
+[[automatic-updates-geoip-processor]]
+=== Automatic updates for the GeoIP processor
+
+By default, {es} now automatically downloads GeoIP2 database updates for the
+`geoip` processor from the Elastic GeoIP endpoint:
+https://geoip.elastic.co/v1/database. You can use the new
+{ref}/geoip-stats-api.html[GeoIP stats API] to get download stats for these
+updates. For more information, see the updated
+{ref}/geoip-processor.html[`geoip` processor documentation].
```

Comment:
FYI @probakowski @martijnvg

Reply:
@jakelandis does this section relate to the geo feature that shouldn't be highlighted in 7.13?

Reply:
@pugnascotia - Actually, it needs to be pulled all together from the 7.13 release notes. We found a last minute issue and will delay this feature by at least 1 release.

Reply:
I raised PR #72743